src/pybind/rbd/rbd.pyx

```diff
@@ -898,13 +898,12 @@ class RBD(object):
     def trash_move(self, ioctx, name, delay=0):
         """
         Move an RBD image to the trash.
-
```

Comment:
Hmm, I'm just running `admin/build-doc` and it reads all the sources and then fails. I'm using Ubuntu Xenial.